packages/fx-core/src/core/FxCore.ts

```diff
@@ -2318,6 +2319,7 @@ export class FxCore {
         ...authParameters,
         authorizationUrl: oauthAuthorizationUrl,
         tokenUrl: oauthTokenUrl,
+        refreshUrl: !oauthRefreshUrl || oauthRefreshUrl === "" ? undefined : oauthRefreshUrl,
```

Comment:
Do you mean `(!oauthRefreshUrl || oauthRefreshUrl === "") ? undefined : oauthRefreshUrl`

And why not use below one:
`oauthRefreshUrl ? oauthRefreshUrl : undefined`